examples/tictactoe/src/constants/tictactoe.ts

```diff
@@ -1,4 +1,4 @@
-export const address = '0x81D8CB015e36922F7ecC5CEAdd018dc7327e6FC7'
+export const address = import.meta.env.TICTACTOE_ADDRESS || '0x81D8CB015e36922F7ecC5CEAdd018dc7327e6FC7'
```

Comment:
If we're going to be supplying the address. Lets instead not have a default provided and simply resort to the cli flag

Is it possible for the process to fail if this flag isn't supplied?

Reply:
yeah its possible make a change